File: java/src/test/java/RomanNumeralsTest.java
import org.junit.Ignore;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class RomanNumeralsTest {

    @Ignore
    @Test
    public void acceptance_test() {
        assertEquals("MMXIX", RomanNumerals.add("CXXIV", "MDCCCXCV"));
    }
}

File: cpp/test/RomanNumeralsTest.cpp

#include <kata/roman_numerals/RomanNumerals.hpp>

#include <string>
#include <gtest/gtest.h>

TEST(RomanNumerals, acceptance_test)
{
	EXPECT_EQ("MMXIX", kata::roman_numerals::add("CXXIV", "MDCCCXCV"));
}
